sailor/pai/__init__.py

```diff
@@ -0,0 +1,4 @@
+
+from .alerts import find_alerts
+
+__all__ = ['find_alerts']
```

Comment:
This comment does not belong here but I want to have it somewhere.

Since `sailor.pai` is a new package, we need to add it to docs/apidoc.rst to be listed in our documentation.

Reply:
:D I thought i changed it.... let me know where you want to have it

Reply:
Just open `docs/apidoc.rst` and you'll know what to do ;-)